system/Log/Handlers/ChromeLoggerHandler.php

```diff
@@ -13,6 +13,7 @@
 
 use CodeIgniter\HTTP\ResponseInterface;
 use Config\Services;
+use Stringable;
```

Comment:
Stringable comes with PHP 8, we can't use it yet. 😢

Reply:
Yes, that's true. But a `use` statement won't error on an unknown class.

Reply:
PHP-CS-Fixer automate `use` statement if set `string|\Stringable`

Reply:
As long as supporting PHP 7.4, we should not use `Stringable`.
It means a normal class `Stringable` and does not make sense.
See https://3v4l.org/TF3Ct